Configuration/Generator/python/TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
```

Comment:
same here
we noticed that this import has a wrong path and failed in production. It should be fixed soon. 
`from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *`

Reply:
> So do I need to create another PR with the following changes with the replacement ---> from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
> ?

Yes, please.

Reply:
I made following changes and recreate PR:
**https://github.com/cms-sw/cmssw/pull/33700**